File: JS Essentials/4.0 For Loops/10. Odd Even Sum.js
function oddEvenSum(arg) {
    let loop = arg.shift()
    let evenSum = 0
    let oddSum = 0

    for (let i = 0; i < loop; i++) {
        let currentNum = +arg.shift()

        if (i & 1) {
            oddSum += currentNum
        } else {
            evenSum += currentNum
        }
    }

    return evenSum == oddSum ? `Yes\nSum = ${evenSum}` : `No\nDiff = ${Math.abs(evenSum - oddSum)}`
}

// console.log(oddEvenSum([4, 10, 50, 60, 20]))

File: Python Essentials/4.0 For Loops/10. Odd Even Sum.py
loop = int(input())
num_list = [int(input()) for _ in range(loop)]
even_sum = sum(num_list[1::2])
odd_sum = sum(num_list[0::2])

print(f'Yes\nSum = {even_sum}' if even_sum == odd_sum else f'No\nDiff = {abs(even_sum - odd_sum)}')


'''
4
10
50
60
20
#Yes
#Sum = 70
'''